### ISSUE TITLE
Option to run Atari programs directly from the console / web ui

### ISSUE THREAD

**Issue by mikrosk** [atarijookie/ce-atari#166]
As a follow-up to #164 -- I don't know about you but sometimes I'm quite frustrated to manually navigate via the Desktop to a tool/game I want to run. When we are already patching Pexec(), file system and whatnot, I guess it can't be a problem to remotely execute an Atari prg via script. I can imagine three modes:
1. mount disk image, reset ST (see #83), play
1. create a virtual desktop.inf with autostart of the given PRG, reset ST, play
1. perhaps abuse CE_DD.PRG with a command to call Pexec() with given PRG path, play

**Comment by atarijookie** [comment]
Yeah, could be done, sounds nice, and will be useful for lazy people like you ;) or those who do the cross-compilation on PC and use ST just for testing the result...

**Comment by miniupnp** [comment]
@mikrosk can you give an example of DESKTOP.INF which autostart the PRG ?
I think you forgot the AUTO/ directory solution.

**Comment by mikrosk** [comment]
@miniupnp on the contrary, you certainly don't want to copy anything into AUTO. How would you start a .TOS app this way?

DESKTOP.INF with autostart can be created from Desktop's app options, at least I hope that it isn't a feature of NEWDESK.INF only.

**Comment by mikrosk** [comment]
@atarijookie think of unit tests, too.

**Comment by miniupnp** [comment]
@mikrosk I guess you can rename to .PRG. Anyway I've no Idea if this is better or not than the DESKTOP.INF solution

**Comment by miniupnp** [comment]
https://github.com/libretro/hatari/blob/4930236a07250b038bc0c9adeb34189401aa7f32/src/tos.c#L326
`"Only TOS versions >= 1.04 support autostarting!\n"`

**Comment by mikrosk** [comment]
Me neither. I guess the .INF / AUTO solution isn't really needed.

**Comment by miniupnp** [comment]
The DESKTOP.INF solution is about adding a line :
`#Z 01 C:\PATH\TO\FILE.TOS@`
(should be 01 for GEM programs, 00 for TOS ones)

**Comment by mikrosk** [comment]
@miniupnp trouble is the .INF solution isn't always _desired_. Imagine running a desktop, having fun and you just want to test one binary. Forcing a reset when you could just do simple Pexec() is pretty lame.
